app/helpers/application_helper.rb

```diff
@@ -72,4 +72,20 @@ def content_tag_if(condition, *args, &block)
       capture(&block)
     end
   end
+
+  def admin_loans_path(*args)
+    # Default to active loans, unless overriden
+    options = { loans: { f: { status_value: ['active'] } } }
+    options = args.pop if args.last.is_a? Hash
```

Comment:
I don't quite follow. For that link, couldn't you do `admin_loans_path(loans: {f: {status_value: ""}})` and the helper would see that that key is defined in the options and _not_ change it to `active`?

I am worried about us going back and forth on this too many times. We need to wrap this up. If my suggestion here doesn't work, please reach out and let's pair on this until it's done.